src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/csip/CSIPMappingResource.kt

```diff
@@ -128,6 +128,55 @@ class CSIPMappingResource(private val mappingService: CSIPMappingService) {
       )
     }
 
+  @PostMapping("/children/all")
```

Comment:
deliberated over a POST or PUT - thought the post was better as we are creating child entries